src/main/java/net/dv8tion/jda/api/Region.java

```diff
@@ -68,12 +61,6 @@
 
     AUTOMATIC("automatic", "Automatic", null, false);
 
-    /**
-     * This {@link java.util.Set Set} represents all regions that can be used for VoiceChannel region overrides.
-     */
-    public static final Set<Region> VOICE_CHANNEL_REGIONS =
```

Comment:
Why is this useless now?

Reply:
Guilds can't have regions anymore

Reply:
sure, but the channels can. Wouldn't this Set still be important for that?

Reply:
kind of, the set was only used to check whether a region is applicable to voice channels. if someone wants to grab available regions, they can just get them from the enum honestly, depends on the point of view

Reply:
@MinnDevelopment opinion on this?

Reply:
I'm fine with removing it